flink-connector-influxdb2/pom.xml

```diff
@@ -38,7 +38,6 @@ under the License.
   <properties>
     <influxdbClient.version>2.0.0</influxdbClient.version>
     <flink.new.version>1.12.2</flink.new.version>
```

Comment:
We should remove flink and scala version here and use the one from the main pom. Maybe merge the current master into our master and then the master into our dev